src/pygeoprocessing/routing/routing.pyx

```diff
@@ -77,12 +77,12 @@ cdef double SQRT2_INV = 1.0 / 1.4142135623730951
 #  321
 #  4x0
 #  567
-cdef int *D8_XOFFSET = [1, 1, 0, -1, -1, -1, 0, 1]
-cdef int *D8_YOFFSET = [0, -1, -1, -1, 0, +1, +1, +1]
+cdef int * D8_XOFFSET = [1, 1, 0, -1, -1, -1, 0, 1]
+cdef int * D8_YOFFSET = [0, -1, -1, -1, 0, +1, +1, +1]

```

Comment:
Was adding this space here intentional? I've always put the * by the variable name when it's a pointer and I think that's fairly standard C++ notation. If not, could you change it back?